tests/Unit/AppInfo/ApplicationTest.php

```diff
@@ -64,4 +69,53 @@ public function dataContainerQuery() {
 	public function testContainerQuery($service, $expected) {
 		$this->assertTrue($this->container->query($service) instanceof $expected);
 	}
+
+	public function testSetupSymfonyEventListeners() {
+		$app = $this->getMockBuilder(Application::class)
```

Comment:
Why are you mocking the class you should be testing? 😕

Reply:
I took this approach, because internally its calling `getContainer`.

Reply:
Since this can't be unittested properly, just test the whole stack: add some notifications in the DB, and trigger the event. Make sure you clean up everything after the test (the listener and the information added in the DB table).

The other option is not to test this method, at least with unittests. Acceptance test should be able to cover the scenario.

Reply:
Updated the test.. Tried to add a user, then added an entry into the notification table. Deleted the user.. Verified that entry is not there in the table.